core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/domain/Concept.java

```diff
@@ -45,28 +47,32 @@ public String getCodeSystem() {
 		return codeSystem;
 	}
 	
-	public String getTerm() {
-		return term;
+	public String getPreferredTerm() {
```

Comment:
Not sure if it is a good idea to call this term a preferred term. If you would like to rename it, then let's use FHIR phrases, like designation or display here. `getDisplay` or `getDisplayTerm` would be alternatives I can think of.